arch/inst/Zicsr/csrrc.yaml

```diff
@@ -23,9 +37,6 @@ access:
   vs: always
   vu: always
 data_independent_timing: false
-pseudoinstructions:
-  - when: xd == 0
-    to: csrc csr,xs1
```

Comment:
There are some inadvertent deletions in your PR from recent commits. Can you rebase to pick these up?